snap/snapcraft.yaml

```diff
@@ -298,6 +300,16 @@ apps:
       - network-bind
       - firewall-control
       - microstack-support
+      
+  file-transfer:
+    command: 'bin/file-transfer-service'
+    daemon: simple
+    restart-condition: on-failure
+    plugs:
+      - network
+      - network-bind
+      - network-control
+      - microstack-support
```

Comment:
Why does the file-transfer app need: `microstack-support` and `network-control`? Both these interfaces are pretty privileged, and I don't see a good reason to grant them.